packages/react-examples/src/react-date-time/DatePicker/DatePicker.Format.Example.tsx

```diff
@@ -14,9 +15,11 @@ const onFormatDate = (date?: Date): string => {
 
 export const DatePickerFormatExample: React.FunctionComponent = () => {
   const [value, setValue] = React.useState<Date | undefined>();
+  const [clearedAnnounced, setClearedAnnounced] = React.useState<JSX.Element | undefined>();
```

Comment:
For this to work properly if the clear button is used multiple times, you'll also need to remove the announced message when a new date is set. Basically instead of passing `setDate` directly to DatePicker's `onSelectDate`, add a wrapper which clears the message too.